codec-http2/src/main/java/io/netty/handler/codec/http2/Http2CodecUtil.java

```diff
@@ -225,7 +225,7 @@ public ChannelPromise doneAllocatingPromises() {
                 doneAllocating = true;
                 if (successfulCount == expectedCount) {
                     promise.setSuccess();
-                    return super.setSuccess();
+                    return setSuccess();
```

Comment:
@nmittler I think the call pattern in either case is:

Top of Stack
1. `SimpleChannelPromiseAggregator.setSuccess(null)`
2. `DefaultChannelPromise.setSuccess()`
3. `SimpleChannelPromiseAggregator.doneAllocatingPromises()`
4. ...

If you intention is to directly call `setSuccess(null)` then lets do that.  This code was done with the assumption that `setSuccess(Void result)` will always be called on any invocation of `setSuccess*` and that is why there is no `SimpleChannelPromiseAggregator.setSuccess()`.